plugins/scaffolder/alpha-api-report.md

```diff
@@ -60,11 +74,64 @@ export const nextSelectedTemplateRouteRef: SubRouteRef<
   PathParams<'/templates/:namespace/:templateName'>
 >;
 
+// @public (undocumented)
+export const registerComponentRouteRef: ExternalRouteRef<undefined, true>;
```

Comment:
Don't think that we should export this. We can just leave the authors to implement an explicit `routeRef` if they need one, or if they don't ignore and don't include the component. This was only done to make it configurable to show a button or not depending on if they bound the route ref in the plugin.